src/Stratis.Bitcoin/IFullNode.cs

```diff
@@ -61,6 +61,14 @@ public interface IFullNode : IDisposable
         /// <param name="failWithError">Set to false to return null instead of throwing an error</param>
         /// <returns></returns>
         T NodeFeature<T>(bool failWithError = false);
+
+        /// <summary>
+        /// Create an instance of the controller type using DI.
```

Comment:
Jesus why? the DI should be in control of instance creation.

Reply:
See below.